free-1.md

```diff
@@ -10,34 +10,84 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+Trying to understand someone else's code&mdash;especially at a large scale&mdash;can be an intimidating, difficult, and overwhelming task,
+even if the person trying to digest the code is an experienced programmer:
+having to read and distinguish keywords in raw source code can be tiring, even if the code is well-commented.
+Documentation can help this process, but words on a page aren't always the best way to express an idea.
+To address this problem, I propose designing a language that translates code into flowcharts,
+which would offer a more visually-appealing representation of code.
+Not only would this solution help anyone (i.e., programmers and non-programmers alike) in understanding code,
+but it could also reduce the effort necessary to produce 'good' documentation,
+and (perhaps optimistically) force programmers to write more high level and well-commented/documented code.
+One really cool potential result would be that people could use skeleton code in the language in which they intend to work
+(and with which, therefore, they _are probably_, or at least _probably should_, be familiar)
+to generate high level flowcharts that express the intended algorithm.
+
```

Comment:
I'm not sure I understand how the language would work, would you write code in this language and then it would be able to be converted into a flow chart? Or would you write code in the DSL alongside your code in a GPL that represents whats happening in the GPL?